core/common/src/main/java/alluxio/wire/LockBlockResult.java

```diff
@@ -105,4 +105,12 @@ public String toString() {
     return Objects.toStringHelper(this).add("lockId", mLockId).add("blockPath", mBlockPath)
         .toString();
   }
+
+  /**
+   * @param result the lock block result
+   * @return true if the block is in cached in Alluxio
+   */
+  public static boolean isBlockCachedInAlluxio(LockBlockResult result) {
+    return result.getLockId() >= 0;
+  }
```

Comment:
This is addressed now. The trick is not to start the method with "is". I was not able to make it work with JsonIgnore.